### ISSUE TITLE
module Mix.Utils is not available

### ISSUE THREAD

**Issue by AvaelKross** [trenpixster/addict#73]
I've installed addict on clean new phoenix app. When trying to log in with incorrect data app responses with '500' error and this log:

```
Request: POST /login
** (exit) an exception was raised:
    ** (UndefinedFunctionError) undefined function Mix.Utils.camelize/1 (module Mix.Utils is not available)
        Mix.Utils.camelize("authentication")
        (addict) lib/addict/controller.ex:137: anonymous fn/1 in Addict.AddictController.return_error/3
        (elixir) lib/enum.ex:1088: Enum."-map/2-lists^map/1-0-"/2
        (addict) lib/addict/controller.ex:136: Addict.AddictController.return_error/3
        (addict) lib/addict/controller.ex:1: Addict.AddictController.action/2
        (addict) lib/addict/controller.ex:1: Addict.AddictController.phoenix_controller_pipeline/2
        (blog_phoenix) lib/phoenix/router.ex:261: BlogPhoenix.Router.dispatch/2
        (blog_phoenix) web/router.ex:1: BlogPhoenix.Router.do_call/2
```

How can I properly handle it?

**Comment by AvaelKross** [comment]
Resolved. I needed to add 'mix' in my project dependencies(in `application` method of mix.exs). It looks completely wrong - add big library only to perform camelizing. I found that it can be safely replaced with `Macro.camelize` which is a part of Elixir by default.

**Comment by trenpixster** [comment]
Closed via #75
